File: demonstracoes/vb.net/NotaSegura/Form1.vb
Public Class Form1

    Private Sub BtnEnviar_Click(sender As Object, e As EventArgs) Handles btnEnviar.Click



        Dim notasegura As New spdNotaSegura.spdNotaSeguraX



        notasegura.Login = txtLogin.Text

        notasegura.Senha = txtSenha.Text

        notasegura.Token = txtToken.Text



        txtXML.Text = notasegura.EnviarXml(txtXML.Text)







    End Sub

End Class

File: demonstracoes/c#/NotaSegura CSHARP/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using spdNotaSegura;
namespace NotaSegura_CSHARP
{
    public partial class Form1 : Form
    {
        spdNotaSeguraX notasegura = new spdNotaSeguraX();
        public Form1()
        {
            InitializeComponent();
            
        }

        private void BtnEnviar_Click(object sender, EventArgs e)
        {

            notasegura.Login = txtLogin.Text;
            notasegura.Senha = txtSenha.Text;
            notasegura.Token = txtToken.Text;

            txtXML.Text = notasegura.EnviarXml(txtXML.Text);

        }
    }
}
